lstchain/reco/reconstructorCC.py

```diff
@@ -227,6 +227,69 @@ def template_interpolation(gain, times, t0, dt, a_hg, a_lg):
     return out
 
 
+@njit(cache=True)
+def nsb_only_waveforms(nsb_waveform, time, is_high_gain, additional_nsb, amplitude, t_0,
+                           t0_template, dt_template, a_hg_template, a_lg_template):
+    """
+    Generate waveforms of pure NSB. NSB photons injected through a fast interpolator using a
+    uniformly sampled base with known origin and step with a certain amplitude at t_0.
```

Comment:
"uniformly sampled base with known origin and step with a certain amplitude at t_0" -> uniformly sampled normalised template with known time of first value and time step. The method requires as input the number of NSB events to inject per pixel, the times of injection and the events normalisations.